components/Post/utils.ts

```diff
@@ -0,0 +1,33 @@
+const getPostDate = (time: number) => {
+  const currentTime = new Date().getTime();
+  const timeDiff = currentTime - time; // in milliseconds
+
+  let timeSincePost = '';
+
+  // 1 minute = 60,000 milliseconds
+  // 1 hour = 3,600,000 milliseconds
+  // 1 day = 86,400,000 milliseconds
+
+  if (timeDiff >= 60000 && timeDiff < 3600000) {
+    timeSincePost = `${Math.round(timeDiff / (1000 * 60))} minutes`;
+  } else if (timeDiff >= 3600000 && timeDiff < 86400000) {
+    timeSincePost = `${Math.round(timeDiff / (1000 * 3600))} hours`;
+  } else if (timeDiff >= 86400000) {
+    timeSincePost = `${Math.round(timeDiff / (1000 * 3600 * 24))} days`;
+  } else {
+    timeSincePost = '1 miunte';
+  }
+
+  return timeSincePost;
+};
+
+const getShortUrl = (url: string) => {
```

Comment:
Lol. 
<img width="799" alt="image" src="https://user-images.githubusercontent.com/60560109/192154427-7aa5ac8b-9a47-41fc-b63f-62f7acb031dd.png">
Take a look at the URL api.